### PR TITLE
FHAC-787 - stack traces at appropriate log levels

### PR DESCRIPTION
**Pull request by christophermay07**
Updates to suppress certain stack traces unless under trace level

### CODE REVIEW

**Comment by christophermay07** [comment]
@alameluchidambaram @vdmehta06 Please review this PR.

File: Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/PurposeOfForDecider.java [hunk 1]
```diff
@@ -104,9 +104,7 @@ private void logPurposeDecision(Boolean purposeFor, String hcid,
             purposeName = "PURPOSE OF";
         }
 
-        LOG.debug("PurposeOfForDecider decision for HCID: " + hcid
-            + " and Service Name: " + serviceName + " is = "
+        LOG.debug("PurposeOfForDecider decision for HCID: " + hcid + " and Service Name: " + serviceName + " is = "
```
Comment (vdmehta06) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/PurposeOfForDecider.java:107]: Optional: if you are creating another commit. Can we use the same logging convention using brackets replacing variable names?

Comment (christophermay07) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/PurposeOfForDecider.java:107]: Sure!  I've been systematically working to replace concatenation with parameterized formatting, but some still slip through.  If I don't need another commit, I'll make a note for the next time we touch this class.

**Comment by alameluchidambaram** [comment]
I have Reviewed this PR and good with merge.

**Comment by vdmehta06** [comment]
I am good with changes and as mentioned that parameterized logging can be taken care in future commit.

**Pull request merged**